extensions/wikia/Wall/Wall.setup.php

```diff
@@ -224,15 +224,21 @@
 $wgGroupPermissions['vstf']['wallarchive'] = true;
 $wgGroupPermissions['helper']['wallarchive'] = true;
 $wgGroupPermissions['sysop']['wallarchive'] = true;
+$wgGroupPermissions['moderator']['wallarchive'] = true;
 
 $wgGroupPermissions['*']['wallremove'] = false;
-$wgGroupPermissions['user']['wallremove'] = true;
+$wgGroupPermissions['user']['wallremove'] = false;
```

Comment:
@jsutterfield - for some reason, I needed to explicitly set it in my development environment - otherwise normal users still had the wallremove permission.